SwiftIsland/Utilities/DataManager.swift

```diff
@@ -87,6 +88,33 @@ extension DataManager: DataManaging {
       }
     }
   }
+  
+  // Todo: Refactor together with get schedule
```

Comment:
Would it be a good idea to merge this with `get<T: Listable>(ofType:completion:)`? Resolving this now will lead to not having to create a new issue while resolving this issue.

Reply:
so I did refactor it to two functions, one for array types (e.g. [Schedule] or [Area]) and one for non-array types (About). I am not quite fully satisfied, because I wanted to merge both to one single function, but I had no idea how to handle a generic result type that can be an array type or a non array type.

I still have to learn more about the use of generics. Maybe somebody else can merge both functions ;)

Reply:
@SpacyRicochet Could you have a look at that? I am unfortunately swamped at the moment.